src/subdomains/generic/user/models/bank-data/bank-data.entity.ts

```diff
@@ -142,4 +143,10 @@ export class BankData extends IEntity {
   get isReviewed(): boolean {
     return this.status === ReviewStatus.COMPLETED;
   }
+
+  get hasMergeComment(): boolean {
+    return [BankDataVerificationError.MERGE_EXPIRED, BankDataVerificationError.MERGE_PENDING].some((e) =>
+      this.comment.includes(e),
+    );
+  }
```

Comment:
Maybe this would be simpler?
```TS
get mergeError(): AmlError | undefined {
  if (this.comment.includes(BankDataVerificationError.MERGE_PENDING)) return AmlError.MERGE_PENDING;
  if (this.comment.includes(BankDataVerificationError.MERGE_EXPIRED)) return AmlError.MERGE_EXPIRED;
}
```